deployment/terraform/visualizer/templates/visualizer_conf.tpl

```diff
@@ -0,0 +1,12 @@
+visualizer {
+    metric-consumer {
+      bootstrap.servers = "localhost:19092"
+      group.id = "my-group"
+      topic = "anomalies"
+      key.deserializer = "org.apache.kafka.common.serialization.StringDeserializer"
+      value.deserializer = "com.expedia.adaptivealerting.kafka.serde.MappedMetricDataJsonSerde$Deser"
+    }
+    elastic-search {
+      host = "localhost"
```

Comment:
Similarly I would expect this to be controlled by a variable, not by a hardcode.

Reply:
Really all of these values should be controlled by variables. The example from `ad-manager_conf.tpl` is much better than `ad-mapper_conf.tpl`.